.github/workflows/prod-build.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Production Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        required: true
+        type: choice
+        description: Which package should be published?
+        options:
+          - vue
+          - angular
+          - react
+      version:
+        required: true
+        type: choice
+        description: Which version should be published?
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+      tag:
+        required: true
+        type: choice
+        description: Which npm tag should this be published to?
+        options:
+          - latest
+          - next
+      preid:
+        type: choice
+        description: Which prerelease identifier should be used? This is only needed when version is "prepatch", "preminor", "premajor", or "prerelease".
+        options:
+          - ''
+          - alpha
+          - beta
+          - rc
+          - next
+
+jobs:
+  dev-build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.dev-build.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        run: npm ci --no-package-lock && lerna bootstrap --ignore-scripts -- --legacy-peer-deps
```

Comment:
May not be necessary, depending on if we need this step - if we do need to run this, do we need the flags on the `lerna bootstrap` command?

Reply:
Well `--ignore-scripts`, where it skips `pre*` steps is currently being used in develop and `--legacy-peer-deps` is ALSO currently being used in the develop build. I don't know the projects well enough to easily tell you if either is necessary or not nor what the impact will be of removing it.

I don't think it's a good idea to make the lerna build + bootstrap scripts different in prod than what they currently are in the develop task.

I will likely want to switch off of lerna and onto pnpm for dependency resolutions and multiple workspace support.

Reply:
Yeah, `--ignore-scripts` prevents puppeteer from running its postinstall step, which shaves ~300-400MB off the `npm install` script